schemaless/create_uuid_map.py

```diff
@@ -17,25 +18,45 @@ def __init__(self):
     def from_files(cls, schemaless_file, uuid_map_file):
         """Build a RecordGraph from schemaless & uuid_map files."""
         rg = cls()
+
+        latest_records = latest_values(schemaless_file)
+
+        # Create a mapping between permit numbers and their associated PPTS
+        # record (so we an ensure they are assigned the same UUID).
+        permit_number_to_ppts = {}
+        for fk, record in latest_records[PPTS.NAME].items():
+            if record['related_building_permits']:
+                for permit_number in record['related_building_permit'].split(","):
+                    #  This assumes that a given permit can only have one PPTS
+                    #  parent. Is that accurate? 
```

Comment:
I suspect that this isn't true

Reply:
easily solvable with a `defaultdict(list)` and `permit_number_to_ppts[permit_number].append(fk)`